xml/System/Array.xml

```diff
@@ -6350,7 +6350,7 @@ int[,,] TDArray = new int[1,1,1];
         <param name="array">The one-dimensional <see cref="T:System.Array" /> to reverse.</param>
         <param name="index">The starting index of the section to reverse.</param>
         <param name="length">The number of elements in the section to reverse.</param>
-        <summary>Reverses the sequence of the elements in a range of elements in the one-dimensional <see cref="T:System.Array" />.</summary>
+        <summary>Reverses the sequence of a subset of the elements in the one-dimensional <see cref="T:System.Array" />.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
```

Comment:
Unrelated to this PR, but I don't think the Remarks here are quite accurate, because the element at myArray[i] may not be changed at all.

Reply:
I think the value of the element at myArray[i] would remain the same if:

1. It's a primitive type element, and...
  1.1. The value is the same as myArray[j], or...
  1.2. i==j, or...
  1.3. myArray.length = 1

Reply:
Or the index `i` is not part of the subset of the array that's being reversed.